libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/ParentChildrenSyncTargetHelper.java

```diff
@@ -60,10 +62,10 @@
 
     public static void saveRecordTreesToLocalStore(SyncManager syncManager, SyncTarget target, ParentInfo parentInfo, ChildrenInfo childrenInfo, JSONArray recordTrees) throws JSONException {
         SmartStore smartStore = syncManager.getSmartStore();
+        SmartSyncSDKManager.getInstance().registerUsedAppFeature(FEATURE_RELATED_RECORDS);
```

Comment:
@wmathurin Is this the best place to add this? I figured this would cover the `SyncUp` and `SyncDown` targets.